src/backend/joanie/core/flows/order.py

```diff
@@ -191,7 +363,7 @@ def failed_payment(self):
         """
 
     @state.on_success()
-    def _post_transition_success(self, descriptor, source, target):  # pylint: disable=unused-argument
+    def _post_transition_success(self, descriptor, source, target, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Why did we add `**kwargs` in the method parameters ?

Reply:
Because I added an argument in a transition, which is passed to `_post_transition_success`.
Without `**kwargs` we got an error:
```
TypeError: OrderFlow._post_transition_success() got an unexpected keyword argument 'billing_address'
```